polaris-patterns/src/configure.ts

```diff
@@ -14,3 +16,34 @@ if (typeof window !== 'undefined') {
 }
 
 export const polarisPatternsVersion = '{{POLARIS_PATTERNS_VERSION}}';
+
+export const DEFAULT_I18N_DETAILS: I18nDetails = {
+  locale: 'en',
+  country: 'CA',
```

Comment:
Ideally, we would avoid using the term `country` wherever possible, due to geopolitics.

```suggestion
  region: 'CA',
```